sql-db/hibernate-fulltext-search/src/test/java/io/quarkus/qe/hibernate/search/AbstractMultitenantNoPanacheIT.java

```diff
@@ -0,0 +1,126 @@
+package io.quarkus.qe.hibernate.search;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.ws.rs.core.Response;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.restassured.common.mapper.TypeRef;
+import io.restassured.http.ContentType;
+
+public abstract class AbstractMultitenantNoPanacheIT {
```

Comment:
Is there any reason why there is the `NoPanache` part in the test? Maybe the Panache tests should rather have the suffix. :)